Tests/UnitTests/Purchasing/Purchases/PurchasesConfiguringTests.swift

```diff
@@ -209,17 +209,6 @@ class PurchasesConfiguringTests: BasePurchasesTests {
         expect(SystemInfo.serverHostURL) == defaultHostURL
     }
 
-    @available(*, deprecated) // Ignore deprecation warnings
-    func testSetDebugLogsEnabledSetsTheCorrectValue() {
-        Logger.logLevel = .warn
-
-        Purchases.debugLogsEnabled = true
-        expect(Logger.logLevel) == .debug
-
-        Purchases.debugLogsEnabled = false
-        expect(Logger.logLevel) == .info
-    }
-
```

Comment:
⚠️  I'm removing this test to fix tests in CI for 2 reasons:
- It's testing a deprecated method
- It's modifying global state, which interferes with every other test. We run tests in parallel, so this is making the new test that requires debug logs to fail randomly, if this test happened to run before (it was changing the  level back to ` .info`).

Reply:
is there any way to
- back up the previous value
- run this atomically
- return the previous value?

Reply:
like, deprecated or not, if we break this, it's still a problem

Reply:
> run this atomically

We would have to block the entire `Logger` from logging anything, and / or block all other tests from running. I can't think of a way to do that.

Reply:
> like, deprecated or not, if we break this, it's still a problem

For sure. I just can't think of a way to test this global state without messing with every other test.

Reply:
Do you have any ideas? :/

Reply:
I do not 😞